tools/ubuntu-setup/ansible.sh

```diff
@@ -3,4 +3,5 @@ sudo apt-add-repository -y ppa:ansible/ansible
 sudo apt-get update -y
 sudo apt-get install -y ansible=2.0.1.0-1ppa~trusty
```

Comment:
because of this

```
Setting up ansible (2.0.1.0-1ppa~trusty) ...
Processing triggers for python-support (1.0.15) ...
Traceback (most recent call last):
  File "/usr/bin/ansible", line 39, in <module>
    from ansible.errors import AnsibleError, AnsibleOptionsError, AnsibleParserError
ImportError: No module named ansible.errors
Adding user travis to group docker
```
https://s3.amazonaws.com/archive.travis-ci.org/jobs/122188733/log.txt

ansible with apt-get can not be used in travis due to some missing libs, so that we have to switch to pip

ansible installation in ansible.sh actually can also be installed via pip